napari/_qt/menus/plugins_menu.py

```diff
@@ -113,9 +117,22 @@ def _add_toggle_widget(*, key=key, hook_type=hook_type):
                 menu.addAction(action)
             action.triggered.connect(_add_toggle_widget)
 
+    def _plugin_manager_dialog_cls(self) -> Union[type, None]:
+        """Return the plugin manager class, if available."""
+        try:
+            # TODO: Register via plugin system?
+            from napari_plugin_manager.qt_plugin_dialog import QtPluginDialog
+        except ImportError as exc:
+            logger.debug("QtPluginDialog not available", exc_info=exc)
```

Comment:
I think that we should use something that will popup to the user interface with a longer description.

Reply:
If we add a popup warning, users that do not want to use the plugin manager will always get a dialog when napari launches. Are we going to require the plugin manager by default in all installs? I had the impression that it would only be a dependency in the constructor bundle, but on the default PyPI/conda-forge reqs.

Reply:
We could have the menu provide a grayed out entry when the plugin manager is not available (e.g. `Plugin manager not available`), but I feel that pollutes the UI unnecessarily?

Reply:
ok. I wrongly understand this code. It looks good.